### DISCUSSION TITLE
MySQL connection UI broken

### DISCUSSION THREAD

**Discussion by ibcht** [dbeaver/dbeaver#19141] [Q&A] [answered]
### Description

When editing / creating a new connection of type MySQL, some elements in the UI are not displayed : SSH, Proxy, SSL.

![image](https://user-images.githubusercontent.com/6569743/220385911-92d29d1a-bb4a-43ca-9d29-3a036e625f95.png)

I have reset the UI settings but this issue is still happening.

When I try the same with another connection type, like MariaDB, it works : 

![image](https://user-images.githubusercontent.com/6569743/220386320-e704c402-e9fd-4881-b5e8-294f73b41eda.png)

The log (which strangely happens in both cases Mysql and MariaDB) : 

`2023-02-21 16:16:08.310 - Service 'org.jkiss.dbeaver.runtime.ui.UIServiceSecurity' not found`

### DBeaver Version

Community Edition 22.3.5.202302171528

### Operating System

Windows 11

### Database and driver

MySQL 8

### Steps to reproduce

1. Set-up a new connection of type MySQL
2. Try to set-up SSH tunnel

**Comment by Destrolaric** [comment]
Could you please send me the log files? You can find instructions here: https://dbeaver.com/docs/wiki/Log-files/. Thank you.

**Comment by E1izabeth** [comment, marked as answer]
Please check that Embedded checkbox is unchecked in Driver settings 
![image](https://user-images.githubusercontent.com/28875055/220392532-e2bf3798-adf7-467f-82e4-e7cadc82463c.png)

If it's unchecked, you could try checking and unchecking it.
Let me know whether it helps.

**Comment by ibcht** [comment]
Oh nice, it works well now ! It must have been a misclick, this was certainly not intended.

Thank you so much Elizabeth, please keep making DBeaver the best.

Irénée
